Source/Core/VideoCommon/RenderBase.h

```diff
@@ -141,7 +161,25 @@ class Renderer
   virtual void ChangeSurface(void* new_surface_handle) {}
   bool UseVertexDepthRange() const;
 
+  // Drawing utility shaders.
+  virtual void DrawUtilityPipeline(const AbstractGraphicsPipeline* pipeline,
```

Comment:
Is it possible these functions are doing too much or abstractions aren't done as well they could be? The parameter lists for both of these are kind of ridiculously large.